### ISSUE TITLE
cargo install-update -al does not update registry

### ISSUE THREAD

**Issue by clemensw** [nabijaczleweli/cargo-update#93]
Some time ago
```
cargo +nightly install-update -al
```
Would print
```
Updating registry ...
```
Before listing the packages.
For a while now it has stopped doing that (or just prints an empty newline) on my installations and does not find updates.  If I install something with cargo
```
cargo +nightly install foo
```
then the registry is updated and install-update finds updated packages.

cargo 1.29.0-nightly
cargo-install-update 1.6.2

I'm not sure if this is related to #15 I only have one directory in `.cargo/registry/index`.

**Comment by nabijaczleweli** [comment]
Ah, fuck; that probably means `cargo-search` @ 1.29.0 doesn't always update registry maybe?

**Comment by nabijaczleweli** [comment]
Indeed that seems the case, and I can't see any other cargo commands that do. Currently in internal discussion with cargo.

**Comment by nabijaczleweli** [comment]
Tracking target: https://github.com/rust-lang/cargo/issues/3377

**Comment by nabijaczleweli** [comment]
New tracking target: https://github.com/rust-lang/cargo/pull/5961

**Comment by nabijaczleweli** [comment]
The takeaway from that is: cargo team don't want people poking around in there (extremely valid, but I will anyway), so they won't be implementing/stabilising `cargo update-registry` – internal [`git fetch refs/heads/master:refs/remotes/origin/master`](https://github.com/rust-lang/cargo/blob/1ee1ef0ea7ab47d657ca675e3b1bd2fcd68b5aab/src/cargo/sources/registry/remote.rs#L204) implementation is the way to go.

**Comment by nabijaczleweli** [comment]
Do please check if current `master` (https://github.com/nabijaczleweli/cargo-update/commit/b7fe3b68c9a18b2e4de50a88653ab24c287ba75a) works?

**Comment by clemensw** [comment]
It now prints out that it's updating the registry so I assume it works.  My installed crates don't update so often so it's awkward to verify that it's detecting updates.

**Comment by nabijaczleweli** [comment]
If it doesn't error out that means it's good :Р

**Comment by nabijaczleweli** [comment]
Released in [`v1.6.3`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v1.6.3).
